src/NuGet.Services.AzureSearch/SearchService/ParsedQuery.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.AzureSearch.SearchService
+{
+    public class ParsedQuery
+    {
+        public ParsedQuery(string text, string packageId)
```

Comment:
Should these have null checks?

Reply:
For text, sure.
`packageId` no.

Reply:
For `text`, looks like `packageId` can be null

Reply:
Ah I missed your comment b/c I didn't reload the page :)

Reply:
Oh, I got that backwards in my words. 😨 

`text` cannot be null.
`packageId` can be null.

Reply:
I retracted my upvote